cpmpy/expressions/core.py

```diff
@@ -131,6 +134,23 @@ def is_bool(self):
     def value(self):
         return None # default
 
+    def _copy_args(self, memodict={}):
```

Comment:
add a one line comment what it does, and also why it exists (versus just being implemented in copy())